tox.ini

```diff
@@ -69,3 +69,10 @@ commands = pip install --ignore-installed 'django>= 1.10, <1.11'
            pip install -r requirements.txt
            pip install mock
            {envpython} setup.py test
+
+[testenv:py36_dj111]
+basepython = python3.6
+commands = pip install --ignore-installed 'django>= 1.11, <1.12'
```

Comment:
Does it not work in Django 2.0?

Reply:
No idea, I just wanted to check whether it does in 1.11 and python 3.6 :)